src/chat/chat_params.rs

```diff
@@ -0,0 +1,289 @@
+                }
+            }
+        }
+    }
+
+    ChatParams = {{ChatParams}} {
+        flow: Overlay,
+        width: Fit,
+        height: Fill,
+
+        main_content = <FadeView> {
+            width: 300
+            height: Fill
+            <View> {
+                width: Fill
+                height: Fill
+                padding: {top: 70, left: 25.0, right: 25.0}
+                spacing: 35
+                flow: Down
+                show_bg: true
+                draw_bg: {
+                    color: #F2F4F7
+                }
+
+                <Label> {
+                    draw_text: {
+                        text_style: <BOLD_FONT>{font_size: 12}
+                        color: #000
+                    }
+                    text: "Inference Parameters"
+                }
+
+                <View> {
+                    flow: Down
+                    spacing: 24
+
+                    temperature = <MoxinSlider> {
+                        default: 1.0
+                        text: "Temperature"
+                        min: 0.0
+                        max: 2.0
+                    }
+
+                    top_p = <MoxinSlider> {
+                        text: "Top P"
+                        min: 0.0
+                        max: 1.0
+                    }
+
+                    <View> {
+                        flow: Right
+                        height: Fit
+                        width: Fill
+                        align: {y: 0.5}
+                        padding: {left: 4}
+                        <Label> {
+                            draw_text: {
+                                text_style: <BOLD_FONT>{font_size: 10},
+                                color: #000
+                            }
+                            text: "Stream"
+                        }
+                        <Filler> {}
+                        stream = <MoxinSwitch> {}
+                    }
+
+                    max_tokens = <MoxinSlider> {
+                        text: "Max Tokens"
+                        min: 100.0
+                        max: 2048.0
+                        step: 1.0
+                    }
+
+                    <View> {
+                        flow: Down
+                        height: Fit
+                        width: Fill
+                        spacing: 12
+                        padding: {left: 4}
+                        <Label> {
+                            draw_text: {
+                                text_style: <BOLD_FONT>{font_size: 10},
+                                color: #000
+                            }
+                            text: "Stop"
+                        }
+                        stop = <MoxinTextInput> {
+                            width: Fill,
+                            // TODO: This should be something like min-height, allowing
+                            // the text input to grow.
+                            height: 65,
```

Comment:
- Without a way to use min-height, the text will overflow after entering some lines of text (like 4 lines).
- We can set this to `Fit` to avoid the issue but that would not match the design.
- We can "mitigate" a bit this by asking for a comma-separated list of terms instead of new line separated list.

Reply:
A comma-separated list of terms sounds good to me as a first implementation. I feel this will be improved having a more complex UI to add each team separately (like adding labels)